tests/ui/asm/named-asm-labels.rs

```diff
@@ -141,7 +153,11 @@ fn main() {
         asm!("{1}: nop", "/* {0} */", const 10, const 20); //~ ERROR avoid using named labels
 
         // Test include_str in asm
-        asm!(include_str!("named-asm-labels.s")); //~ ERROR avoid using named labels
+        asm!(include_str!("named-asm-labels.s"));
+        //~^ ERROR avoid using named labels
+        //~^^ ERROR avoid using named labels
+        //~^^^ ERROR avoid using named labels
+        //~^^^^ ERROR avoid using named labels
```

Comment:
For future reference, this can also be written as
```suggestion
        //~^ ERROR avoid using named labels
        //~| ERROR avoid using named labels
        //~| ERROR avoid using named labels
        //~| ERROR avoid using named labels
```